src/bootstrap/build/sanity.rs

```diff
@@ -78,6 +79,24 @@ pub fn check(build: &mut Build) {
             panic!("the iOS target is only supported on OSX");
         }
 
+        if cfg!(windows) {
```

Comment:
Isn't this needed for running `llvm-config` at all? Or running the compiler if it's dynamically linked? I may be misunderstanding the purpose for this though?